src/popup/App.vue

```diff
@@ -218,6 +222,15 @@ export default defineComponent({
       }
     });
 
+    // Redirect to account page if no multisig accounts are present
+    // only if current page is a multisig page
+    watch(multisigAccounts,
+      (value) => {
+        if (!value?.length && route.meta.isMultisig) {
```

Comment:
```suggestion
        if (!value?.length && (route.meta as WalletRouteMeta).isMultisig) {
```

It will help with some future refactors in the future.